setup.cfg

```diff
@@ -41,7 +41,7 @@ test = pytest
 
 [tool:pytest]
 collect_ignore = ["setup.py"]
-addopts = --verbose tests
+addopts = --verbose
```

Comment:
This setup previously prevented developers from running tests selectively. Now, tests can be run individually or by module.

Reply:
The other workaround is to use `pytest -c doesnotexist tests/test_only_me.py`